tpke/src/ciphertext.rs

```diff
@@ -57,7 +55,7 @@ impl<E: Pairing> Ciphertext<E> {
 }
 
 pub fn encrypt<E: Pairing>(
-    message: &[u8],
+    message: SecretBox<Vec<u8>>,
```

Comment:
Maybe it's worth making a type for a secret-boxed-slice to avoid double allocation. Not for this PR I guess.